Dockerfile

```diff
@@ -19,13 +19,16 @@ ADD mix.exs mix.lock /app/
 ADD config /app/config
 WORKDIR /app
 
-RUN mix deps.get --only prod
+RUN mix deps.get
 RUN mix deps.compile
 
 ADD . /app
 RUN mix compile
 RUN mix phx.digest
 
+RUN yarnpkg install --no-progress
+RUN yarnpkg deploy
```

Comment:
We should add yet another stage in the file, using the same image we're running for the `webpack` service right now, and build the UI as we're now doing in the `build.sh` file - but in this Dockerfile, so we can then `COPY --from=webpack` those resulting files.